bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,47 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
```

Comment:
variable names are not good ;P and can you please explain what exactly are you doing.

Reply:
what do you suggest?

Reply:
what constitutes a good variable name?

Reply:
the name which is well suited for what it does and not so long and from it's name we can have a idea that what value it is holding(I guess 😉 )